src/components/CallbackForm/CallbackForm.js

```diff
@@ -168,9 +174,23 @@ export default function CallbackForm({
         }
         if (callTypes.includes(value)) {
             setHelpNeeded(value);
+            setSubtypeDropdownValues(value);
         }
     };
 
+    const setSubtypeDropdownValues = (value) => {
```

Comment:
I would possibly change this from 'value' to something descriptive, although I can see its purpose in the next line.